internal/util/util.go

```diff
@@ -32,6 +34,7 @@ func LookupImage(ctx *types.SystemContext, store storage.Store, image string) (*
 		return nil, err
 	}
 	return localImage, nil
+	//
```

Comment:
What is this?  Looks like it was supposed to be a comment?  Remove if not necessary.